apps/openmw/mwmechanics/actors.cpp

```diff
@@ -374,9 +374,13 @@ namespace MWMechanics
             if (LOS)
             {
                 MWBase::Environment::get().getMechanicsManager()->startCombat(actor1, actor2);
-                if (!againstPlayer) // start combat between each other
+                if (!againstPlayer) // start combat between each other if engaging actor is not one of the player followers
```

Comment:
This is done to prevent not-committing the crime when startcombat is issued i.e. when I issue "startcombat elone" on arrille, he summons the skeleton and elone starts fighting it without skeleton commiting the crime